src/contentScripts/index.ts

```diff
@@ -1,22 +1,23 @@
+import log from "loglevel";
 import { browser } from "webextension-polyfill-ts";
-import { ActionType as IdentityActionType } from "@src/ui/ducks/identities";
+
+import { ReduxAction } from "@src/types";
 import { ActionType as AppActionType } from "@src/ui/ducks/app";
-import log from "loglevel";
+import { ActionType as IdentityActionType } from "@src/ui/ducks/identities";
 
-(async function () {
-  try {
-    const url = browser.runtime.getURL("js/injected.js");
-    const container = document.head || document.documentElement;
-    const scriptTag = document.createElement("script");
-    scriptTag.src = url;
-    scriptTag.setAttribute("async", "false");
-    container.insertBefore(scriptTag, container.children[0]);
-    container.removeChild(scriptTag);
+try {
+  const url = browser.runtime.getURL("js/injected.js");
+  const container = document.head || document.documentElement;
+  const scriptTag = document.createElement("script");
+  scriptTag.src = url;
+  scriptTag.setAttribute("async", "false");
+  container.insertBefore(scriptTag, container.children[0]);
+  container.removeChild(scriptTag);
 
-    window.addEventListener("message", async (event) => {
-      const { data } = event;
-      if (data && data.target === "injected-contentscript") {
-        const res = await browser.runtime.sendMessage(data.message);
+  window.addEventListener("message", (event: MessageEvent<{ target: string; message: unknown; nonce: string }>) => {
```

Comment:
Could we move this type better in the `@src/types/index.ts`?

Reply:
I'll fix